doc/source/tech_note/Methane/CLM50_Tech_Note_Methane.rst

```diff
@@ -426,7 +426,7 @@ molecular free-air diffusion coefficients (:math:`{D}_{0}`
 .. table:: Temperature dependence of aqueous and gaseous diffusion coefficients for CH\ :sub:`4` and O\ :sub:`2`
 
  +----------------------------------------------------------+----------------------------------------------------------+--------------------------------------------------------+
- | :math:`{D}_{0}` (m\ :sup:`2` s\ :sup:`-1`)               | CH\ :sub:`4`                                             | O\ :sub:`2`                                            |
+ | :math:`{D}_{0}` (cm\ :sup:`2` s\ :sup:`-1`)              | CH\ :sub:`4`                                             | O\ :sub:`2`                                            |
```

Comment:
I'm assuming you checked these units with the code?

Reply:
Yep! The constants are set here, where the correct units are given in the comment:
https://github.com/samsrabin/CTSM/blob/7ae1372c0c39747ba71ae399cd2c5964e72f917d/src/main/clm_varcon.F90#L244-L247

When they're used, they're multiplied by 1e-4 to convert `D0` from cm2/s to m2/s:
https://github.com/samsrabin/CTSM/blob/7ae1372c0c39747ba71ae399cd2c5964e72f917d/src/soilbiogeochem/SoilBiogeochemNitrifDenitrifMod.F90#L400

However, it looks like the table is missing a digit. Will add.